index.js

```diff
@@ -1,13 +1,41 @@
-const mongoose = require('mongoose');
-const Recipe = require('./models/Recipe.model'); // Import of the model Recipe from './models/Recipe.model.js'
-const data = require('./data.js'); // Import of the data from './data.js'
+const mongoose = require("mongoose");
+const Recipe = require("./models/Recipe.model"); // Import of the model Recipe from './models/Recipe.model.js'
+const data = require("./data.js"); // Import of the data from './data.js'
+
+/* Recipe.create({
+  title: "tortilla de patatas",
+  level: "Easy Peasy",
+  ingredients: ["potatos", "eggs", "salt", "garlic", "onions"],
+  cuisine: "spanish",
+  dishType: "Dish",
+  image:
+    "https://imagenes.heraldo.es/files/image_990_v1/uploads/imagenes/2018/06/18/_unnamed_796d3acc.jpg",
+  duration: 20,
+  creator: "Lluís Mañas",
+  created: 1 / 30 / 2020
+}).then(x => console.log(recipe.title)); */
 
 // Connection to the database "recipeApp"
 mongoose
```

Comment:
i woud have this at the top directly on line 5. Then write all your following code.